contracts/priv/unity.yml

```diff
@@ -1441,6 +1460,42 @@ components:
       required:
         - id
         - name
+    Cluster:
+      properties:
+        isAvailable:
+          description: determines if the cluster is available
+          nullable: false
+          readOnly: true
+          type: string
+        isBeta:
+          description: determines if the cluster is in beta
+          nullable: false
+          readOnly: true
+          type: string
+        priority:
+          description: priority for sorting the cluster
+          nullable: false
+          readOnly: true
+          type: string
+        providerId:
+          description: the id of the cloud provider
+          nullable: false
+          readOnly: true
+          type: string
```

Comment:
Should we have `providerName` in here as well?

Reply:
We could, I just didn't return anything that we weren't currently using, and the way we render the different providers in the UI includes other things that can't come from the API anyway, like the different logos.

Reply:
We don't actually use the provider name anywhere in the UI? That surprises me a bit, but if that's the case, then I'm fine with this as-is.

Reply:
Actually you're completely correct, we do return the provider name, I was only looking at the `extract_region` function. Good catch.